src/MasterSite.php

```diff
@@ -325,6 +325,52 @@ public function clean_post_cache( $post_id, $post_after, $post_before ) {
 		if ( defined( 'DOING_AUTOSAVE' ) && DOING_AUTOSAVE ) {
 			return;
 		}
+
+		// Flush TAB posts cache, if the selected act page status change.
+		$parent_act_id = (int) planet4_get_option( 'act_page' );
+		if ( 'page' === $post_after->post_type && $parent_act_id === $post_after->post_parent ) {
+
+			// Flush cache only when a page status changes from publish to any non-public status & vice versa.
+			if (
+				( 'publish' === $post_before->post_status && 'publish' !== $post_after->post_status ) ||
+				( 'publish' === $post_after->post_status && 'publish' !== $post_before->post_status )
+			) {
+				global $wpdb, $nginx_purger;
+
+				$tab_posts = [];
```

Comment:
We probably use it in other places in the code, but it's kinda confusing that it abbreviates to an existing and common word `tab`. Could be less ambiguous if we refer to it as `$boxout_posts`?

Reply:
Then we're good until we have a second kind of boxout :smile: